### PR TITLE
chore: intro `editorconfig-maven-plugin` for verifying code style defined in `.editorconfig`

### PR DESCRIPTION
**Pull request by HJ-Young**
refer to [PR#2591](https://github.com/apache/incubator-hugegraph/pull/2591)
- check code style during the verify phase of the build
- use mvn editorconfig:format for one-click code formatting

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -540,6 +540,37 @@
...
+                <configuration>
+                    <!-- See http://ec4j.github.io/editorconfig-maven-plugin/ for full configuration reference -->
+                    <excludes>
+                        <!-- Note that maven submodule directories and many non-source file patterns are excluded by default -->
+                        <!-- see https://github.com/ec4j/editorconfig-linters/blob/master/editorconfig-lint-api/src/main/java/org/ec4j/lint/api/Constants.java#L37 -->
+                        <!-- You can exclude further files from processing: -->
+                        <exclude>**/*.txt</exclude>
+                        <exclude>**/.flattened-pom.xml</exclude>
+                        <exclude>**/hubble-fe/**/*</exclude>
+                        <exclude>**/apache-hugegraph-hubble-incubating-1.3.0/**/*</exclude>
```
Comment (VGalaxies) [inline pom.xml:565]:
```suggestion
                        <exclude>**/apache-hugegraph-hubble-incubating-*/**/*</exclude>
```

File: .idea/vcs.xml
Comment (VGalaxies) [inline .idea/vcs.xml:1]: why removed license header

File: .github/workflows/tools-ci.yml [hunk 2]
```diff
@@ -48,6 +48,9 @@ jobs:
         run: |
           cp $HOME/.m2/settings.xml /tmp/settings.xml
           mv -vf .github/configs/settings.xml $HOME/.m2/settings.xml
+      - name: Editorconfig
+        run: |
+          mvn editorconfig:format
```
Comment (VGalaxies) [inline .github/workflows/tools-ci.yml:53]: useless?

Comment (HJ-Young) [reply .github/workflows/tools-ci.yml:53]: true, has been deleted

**Review by VGalaxies — APPROVED** [review]

File: hugegraph-hubble/hubble-be/src/main/java/org/apache/hugegraph/exception/GenericException.java
Comment (imbajin) [inline hugegraph-hubble/hubble-be/src/main/java/org/apache/hugegraph/exception/GenericException.java:1]: file separator diff?

like `CRLF -> LF`?

**Review by imbajin — APPROVED** [review]

File: hugegraph-hubble/hubble-be/src/main/java/org/apache/hugegraph/exception/GenericException.java
Comment (VGalaxies) [reply hugegraph-hubble/hubble-be/src/main/java/org/apache/hugegraph/exception/GenericException.java:1]: indeed so

**Pull request merged**